infrastructure/lib/constructs/database.ts

```diff
@@ -0,0 +1,260 @@
+          afterBundling(inputDir: string, outputDir: string): string[] {
+            return [
+              `curl -sL -o /tmp/rds-ca-2017.pem https://truststore.pki.us-gov-west-1.rds.amazonaws.com/global/global-bundle.pem`,
+              `cp /tmp/rds-ca-2017.pem ${outputDir}/rds-gov-ca-bundle-2017.pem`,
+            ];
+          },
+          beforeInstall() {
+            return [];
+          },
+        },
+      },
+    });
+    this.bootstrapResource = new cdk.CustomResource(this, "CustomResource", {
+      serviceToken: handler.functionArn,
+      properties: {
+        DatabaseName: props.databaseName,
+        DatabaseHost: props.cluster.clusterEndpoint.hostname,
+        DatabaseSecretName: props.secretName,
+      },
+    });
+    this.backingLambda = handler;
+  }
+}
+
+/**
+ * A highly-opinionated RDS configuration.
+ *
+ * Builds an Aurora PostgreSQL database with various security settings as well as
+ * automatic bootstrapping of an underlying database.
+ */
+export class Database extends cdk.Construct {
+  /**
+   * The underlying Aurora cluster.
+   */
+  public readonly cluster: rds.DatabaseCluster;
+  /**
+   * The Secret resource for the Aurora cluster's master user.
+   */
+  public readonly adminSecret: secretsmanager.ISecret;
+  /**
+   * A way to identify whether the database has actually been created on the Aurora host.
+   *
+   * Any resource that needs to connect to the database for any operation should depend
+   * directly on this. It is not guaranteed that the database or users will be created and
+   * properly configured prior to this resource being complete.
+   */
+  // Using a ConcreteDependable means that we can change the underlying definition of what it
+  // means for this to be "ready" without having to update the API. We just need to add/change
+  // resources in the ConcreteDependable.
+  public readonly databaseReady: cdk.ConcreteDependable;
+
+  public readonly clusterResourceId: string;
+
+  public readonly databaseName: string;
+
+  constructor(scope: cdk.Construct, id: string, props: DatabaseProps) {
+    super(scope, id);
+
+    this.databaseName = props.databaseName;
+
+    const dbEngine = rds.DatabaseClusterEngine.auroraPostgres({ version: rds.AuroraPostgresEngineVersion.VER_13_4 });
+    const parameters = new rds.ParameterGroup(this, "PostgresParams", {
+      engine: dbEngine,
+      parameters: {
+        "rds.force_ssl": "1",
+      },
+    });
+    const cluster = new rds.DatabaseCluster(this, "Database", {
+      engine: dbEngine,
+      instanceProps: {
+        vpc: props.vpc,
+        // The cheapest available instance type for the engine we've chosen.
+        instanceType: ec2.InstanceType.of(ec2.InstanceClass.T3, ec2.InstanceSize.LARGE),
+        allowMajorVersionUpgrade: true,
+        autoMinorVersionUpgrade: true,
+        deleteAutomatedBackups: false,
+        // Performance Insights are not available in AWS GovCloud (US)
+        // enablePerformanceInsights: true,
```

Comment:
So I want to explain _why_ we're not using a feature that I'd like to see us using. This felt like the best way possible to do that; do you have a recommendation on how to document that? Maybe Confluence?

Reply:
I think this is a good example of when a comment is warranted.  It's almost a TODO, but it can't be done yet because we're waiting for that feature set to reach GovCloud.  Maybe something like _"Would like to enable XYZ when available in GovCloud..."_